### PR TITLE
[DOCS] Reformat requirements callouts in "Endpoint management"

### LINKED ISSUE #2614: [DOCS] Reformat requirements callouts in "Endpoint management"
## Description 

Reformat requirements information for Endpoint artifacts/features in the **[Endpoint management](https://www.elastic.co/guide/en/security/master/sec-manage-intro.html)** section.

Use the sidebar requirements sidebar format tested out in https://github.com/elastic/security-docs/pull/2536:

image

## Topics to update

- [Endpoints](https://www.elastic.co/guide/en/security/master/admin-page-ov.html)
- [Host isolation](https://www.elastic.co/guide/en/security/master/host-isolation-ov.html) ??
- [Trusted applications](https://www.elastic.co/guide/en/security/master/trusted-apps-ov.html)
- [Event filters](https://www.elastic.co/guide/en/security/master/event-filters.html)
- [Host isolation exceptions](https://www.elastic.co/guide/en/security/master/host-isolation-exceptions.html)
- [Blocklist](https://www.elastic.co/guide/en/security/master/blocklist.html)

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2614.

### Previews:

- [Endpoints](https://security-docs_2617.docs-preview.app.elstc.co/guide/en/security/master/admin-page-ov.html)
- [Host isolation](https://security-docs_2617.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-ov.html)
- [Trusted applications](https://security-docs_2617.docs-preview.app.elstc.co/guide/en/security/master/trusted-apps-ov.html)
- [Event filters](https://security-docs_2617.docs-preview.app.elstc.co/guide/en/security/master/event-filters.html)
- [Host isolation exceptions](https://security-docs_2617.docs-preview.app.elstc.co/guide/en/security/master/host-isolation-exceptions.html)
- [Blocklist](https://security-docs_2617.docs-preview.app.elstc.co/guide/en/security/master/blocklist.html)

### CODE REVIEW

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by joepeeples** [comment]
cla/check

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 1]
```diff
@@ -6,21 +6,22 @@ Host isolation allows you to isolate hosts from your network, blocking communica
...
-* AWS Linux 2
-
-To isolate and release hosts in any operating system, you must have the built-in `superuser` role. For more information, refer to {ref}/built-in-users.html[Built-in users].
-=========================
+.Requirements
+[sidebar]
+--
+* Host isolation is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature. 
+
+* For {stack} version >= 7.15.0, host isolation is supported for endpoints running Windows, macOS, and these Linux distributions:
```
Comment (benironside) [inline docs/management/admin/host-isolation-ov.asciidoc:14]:
```suggestion
* For {stack} versions >= 7.15.0, host isolation is supported for endpoints running Windows, macOS, and these Linux distributions:
```
Not 100% about this, but I would this sentence as "for stack versions greater than or equal to..." so I think the plural fits

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 2]
```diff
@@ -6,21 +6,22 @@ Host isolation allows you to isolate hosts from your network, blocking communica
...
+
+** CentOS/RHEL 8
+** Debian 11
+** Ubuntu 18.04
```
Comment (benironside) [inline docs/management/admin/host-isolation-ov.asciidoc:18]: Could probably combine the three Ubuntu lines into one, e.g. "Ubuntu 18.04, 20.04, and 22.04"

**Review by benironside — COMMENTED** [review]
Looks good, just left a couple minor suggestions

File: docs/management/admin/host-isolation-ov.asciidoc [hunk 3]
```diff
@@ -6,21 +6,22 @@ Host isolation allows you to isolate hosts from your network, blocking communica
...
+** Ubuntu 20.04
+** Ubuntu 22.04
```
Comment (joepeeples) [inline docs/management/admin/host-isolation-ov.asciidoc:20]:
```suggestion
** Ubuntu 18.04, 20.04, and 22.04
```

**Review by nastasha-solomon — APPROVED** [review]
🚢 it!

**Review by jmikell821 — APPROVED** [review]
Loving the frequent use of the sidebar - LGTM!

**Pull request merged**